### DISCUSSION TITLE
Freezing an object in multiple frames

### DISCUSSION THREAD

**Discussion by Igasko** [keiwando/toonsquid#530] [Q&A]
Hi, my girlfriend is using toonsquid and she has this problem where she has a character of the witch and its on different positions in different layers, is there a way to like "freeze" this position or copy it trough multiple frames?
I would be really happy if someone had a video because from instructions from the internet we couldnt really figure much out.

**Comment by kavlor** [comment]
I think you need to elaborate because it's hard to  know exactly what you mean. Do you have keyframes, is it  a rig . Are you talking about frame hold which can be done by pulling the handles that appear when you tap a frame.
